fontc_crater/src/ttx_diff_runner.rs

```diff
@@ -43,11 +48,9 @@ pub(super) fn run_ttx_diff(source: &Path) -> RunResult<DiffOutput, DiffError> {
             }
             Ok(RawDiffOutput::Error(error)) => RunResult::Fail(DiffError::CompileFailed(error)),
         },
+        Some(124) => RunResult::Fail(DiffError::Other("ttx_diff timed out".to_string())),
```

Comment:
Nit: seems worth a dedicated error code?

Reply:
this should be uncommon and adding a new category requires modifying a bunch of other code (like the html generation would need to have a new section etc)